docker-compose.yml

```diff
@@ -37,3 +38,12 @@ services:
       image: postgres:10
       ports:
         - 5432:5432
+
+  postgres_test_database:
+    <<: *env_file
+    container_name: postgres_test_database
+    volumes:
+      - postgres_test_database_volume:/var/lib/postgresql/data/
+    image: postgres:10
```

Comment:
Some newer one could be used. postgres:11 is available; version 12 might not be available yet.